configuration/uart.md

```diff
@@ -1,51 +1,111 @@
-# The Raspberry Pi UARTs
+# UART configuration
 
-The SoCs used on the Raspberry Pis have two built-in UARTs, a [PL011](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0183g/index.html) and a mini UART. They are implemented using different hardware blocks, so they have slightly different characteristics. However, both are 3.3V devices, which means extra care must be taken when connecting up to an RS232 or other system that utilises different voltage levels. An adapter must be used to convert the voltage levels between the two protocols. Alternatively, 3.3V USB UART adapters can be purchased for very low prices. 
- 
-By default, on Raspberry Pis equipped with the wireless/Bluetooth module (Raspberry Pi 3 and Raspberry Pi Zero W), the PL011 UART is connected to the Bluetooth module, while the mini UART is used as the primary UART and will have a Linux console on it. On all other models, the PL011 is used as the primary UART. 
+There are two types of UART available on the Raspberry Pi -  [PL011](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0183g/index.html) and mini UART. The PL011 is a capable, broadly 16550-compatible UART, while the mini UART has a reduced feature set.
```

Comment:
I wasn't aware that the PL011 design is based on 16550, and I don't think this factoid adds anything - why mention this and not that the mini-UART is an 8250 clone?

Reply:
That sentence does not claim the design is based on the 16550. The Broadcom datasheet for the 2835 also compares it to the 16550 in a similar way. Comparison to the 16550 is used as a way of saying that it is fairly standard in terms of its capability. If you have a suggestion for a better description I'm happy to use that instead.